modules/candidate_list/README.md

```diff
@@ -28,17 +28,17 @@ can only see candidates at study sites where they are affiliated.
 
 ## Configurations
 
-The "useProjects" configuration affects whether the table has a
+The `useProjects` configuration affects whether the table has a
 "Project" column for the candidate, and whether there is a filter
 dropdown for that column.
 
-The "useEDC" configuration variable has a similar function for the
-EDC column.
+The `useEDC` configuration variable has a similar function for the
+"EDC" column.
```

Comment:
Shouldn't it be ` instead of " to be consistent?

Reply:
I think it is fine, ` is more for variable names.